vm/class.go

```diff
@@ -1545,6 +1572,38 @@ var builtinClassCommonInstanceMethods = []*BuiltinMethodObject{
 
 		},
 	},
+	// Just evaluates a given block with the receiver and returns the receiver.
+	// #tap method literally "taps into" the method chain and
+	// good for inspecting method chains.
+	// Any arguments to the method are just ignored.
+	//
+	// ```ruby
+	// (1..10).tap do |x|
```

Comment:
can also put

```ruby
a = (1..10)
a.tap do |x|
# omit the rest
puts(a) #=> (1..10)
```

to show the original object is not touched

Reply:
Got it